.github/workflows/test.yml

```diff
@@ -20,7 +20,6 @@ jobs:
           - ubuntu-18.04
           - macos-latest
         python:
-          - 3.6  # will also be used to test with _DANDI_LOG_GIRDER set
```

Comment:
The workflow step for setting `DANDI_LOG_GIRDER` should be updated to trigger on 3.7 instead of 3.6.

Reply:
I have pushed update for all the 3.6 --> 3.7